kayenta-core/src/main/java/com/netflix/kayenta/metrics/SynchronousQueryProcessor.java

```diff
@@ -88,10 +88,8 @@ public String executeQuery(
             metricsService.queryMetrics(
                 metricsAccountName, canaryConfig, canaryMetricConfig, canaryScope);
         success = true;
-      } catch (RetrofitError e) {
-
-        boolean retryable = isRetryable(e);
-        if (retryable) {
+      } catch (SpinnakerHttpException e) {
+        if (e.getRetryable()) {
```

Comment:
Pretty tempting to use [RetrySupport](https://github.com/spinnaker/kork/blob/44281f9c707997b9942185f625637f1efb9bfcc6/kork-exceptions/src/main/java/com/netflix/spinnaker/kork/core/RetrySupport.java) here.

Reply:
I agree but I think it makes more sense to save that for a follow-up PR since it's separate from replacing instances of `RetrofitError`